manifests/charts/cloudcore/templates/deployment_controllermanager.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   annotations: {{- toYaml . | nindent 4 }}
   {{- end }}
   name: kubeedge-controller-manager
-  namespace: kubeedge
+  namespace: {{ .Release.Namespace }}
```

Comment:
i can not find where Release.Namespace come from?

Reply:
It comes from the namespace where the chart was released to.

Reply:
can you point it out in source codes? thanks

Reply:
Is the document ok? https://helm.sh/docs/topics/charts/#predefined-values
<img width="977" alt="image" src="https://user-images.githubusercontent.com/76980726/197663168-986172ac-a888-4bcb-97c8-5ca0b1ceb375.png">

Reply:
lgtm
/cc @zhu733756 @zc2638